src/main/frontend/model/samples/check-hardcoded-urls.json

```diff
@@ -0,0 +1,70 @@
+{
+	"logLevel": "info",
+	"hops": [
+		{
+			"type": "declare",
+			"declarations": {
+				"path": "\"/content/swisscom/de\"",
+				"phrase": "\"https://www.swisscom.ch/content/de\"",
+				"extension": "\".html\""
+			}
+		},
+		{
+			"type": "runScript",
+			"code": "xPathQuery = str:join([\"/jcr:root\", path, \"//*[jcr:contains(., '\", phrase, \"') and jcr:contains(., '\", extension, \"') ]\"]);\n\n",
+			"extension": "jexl",
+			"putLocalsBackIntoScope": true
+		},
+		{
+			"type": "nodeQuery",
+			"query": "${xPathQuery}",
+			"queryType": "xpath",
+			"hops": [
+				{
+					"type": "each",
+					"expression": "node.properties",
+					"hops": [
+						{
+							"type": "filterNode",
+							"expression": "property.type == 1 && !property.multiple && str:contains(jcr:val(node, property.name), phrase ) && str:contains(jcr:val(node, property.name), extension) ",
```

Comment:
Maybe we should add an additional helper in the `jcr:` namespace that would allow checking property types more expressively, e.g. `jcr:isString(property)`.